datasafe-directory/datasafe-directory-impl/src/main/java/de/adorsys/datasafe/directory/impl/profile/config/DefaultUserProfileLocationImpl.java

```diff
@@ -18,14 +21,13 @@ public DefaultUserProfileLocationImpl(Uri systemRoot) {
 
     public AbsoluteLocation<PrivateResource> locatePrivateProfile(UserID ofUser) {
         return new AbsoluteLocation<>(
-                new BasePrivateResource(PRIVATE_PROFILE.resolve(ofUser.getValue())).resolveFrom(dfsRoot())
```

Comment:
Looks like currently this will cause more problems for us - lets keep it like it was, so tests and expectations does not need to be changed